### PR TITLE
Setupfolders takes global flag, moved from main to commands, creates ~/.config/fontman folder.

### PR DESCRIPTION
**Pull request by meowox**

### CODE REVIEW

**Comment by meowox** [comment]
This also resolves #8. I forgot to mention in 06f4381b82aad2d374d70d138b5d8cfe65587772 that SetupFolders now sets up the .config/fontman/ folder as well.

File: cmd/fontman/commands/install.go [hunk 1]
```diff
@@ -2,26 +2,37 @@ package commands
...
 	fmt.Println("Installing font(s)...")
 
 	return nil
 }
 
 // Constructs the 'install' subcommand.
 func RegisterInstall() *cli.Command {
+	// TODO: style/ex_style should be arrays of strings; look into how the lib handles multi-parameter argument
 	var style string
+	var ex_style string
```
Comment (gmisail) [inline cmd/fontman/commands/install.go]: I think this should be `exStyle` or `excludeStyle` just to maintain consistency.

File: cmd/fontman/commands/uninstall.go [hunk 2]
```diff
@@ -16,12 +23,15 @@ func onUninstall(c *cli.Context) error {
 // Constructs the 'uninstall' subcommand.
 func RegisterUninstall() *cli.Command {
 	var style string
+	var ex_style string
```
Comment (gmisail) [inline cmd/fontman/commands/uninstall.go]: Same thing as above, I think we should stick with camelCase if possible (I prefer pascal_case, but Go likes camelCase 🤷‍♂️)

**Review by gmisail — CHANGES_REQUESTED** [review]
Everything looks good, just some minor variable name changes.

**Pull request merged**